rtv/submission.py

```diff
@@ -91,19 +51,24 @@ def toggle_comment(self):
             # cursor index to go out of bounds.
             self.nav.page_index, self.nav.cursor_index = current_index, 0
 
-    def refresh_content(self):
+    @SubmissionController.register(curses.KEY_LEFT, 'h')
+    def exit_submission(self):
+        self.active = False
 
+    @SubmissionController.register(curses.KEY_F5, 'r')
+    def refresh_content(self):
         url = self.content.name
         self.content = SubmissionContent.from_url(self.reddit, url, self.loader)
         self.nav = Navigator(self.content.get, page_index=-1)
 
+    @SubmissionController.register(curses.KEY_ENTER, 10, 'o')
```

Comment:
It's the ascii value of the enter key. The curses.KEY_ENTER wasn't working. I checked the curses documentation and it says KEY_ENTER is unreliable, so I added 10 as a fallback.